web/public/js/directives/cd-action-bar/index.js

```diff
@@ -6,7 +6,8 @@
     .component('cdActionBar', {
       bindings: {
         open: '<',
-        forceFixed: '<'
+        forceFixed: '<',
+        showOverflowButton: '<'
```

Comment:
I had this in thought in the first place, but the realisation is way bigger than a boolean :p